script/foundry/utils/DeployHelper.sol

```diff
@@ -0,0 +1,356 @@
+/* solhint-disable no-console */
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+// external
+import { UpgradeableBeacon } from "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { console2 } from "forge-std/console2.sol";
+import { Script } from "forge-std/Script.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+// TODO: fix the install of this plugin for safer deployments
```

Comment:
We can remove the TODO since we have StorageLayoutChecker

Reply:
[a4c995e](https://github.com/storyprotocol/protocol-core-v1/pull/36/commits/a4c995e042cf8362ae9dd0ed8835eaa9c9febe3d)